### ISSUE TITLE
Icons in hyperlinks are not audible accessible

### ISSUE THREAD

**Issue by blackfalcon** [AlaskaAirlines/auro-hyperlink#134]
## Describe the bug

Icons in the hyperlink are not being read aloud by screen readers. 

## To Reproduce

1. Go to https://auro.alaskaair.com/components/auro/hyperlink
1. Turn on screen reader
1. See error

## Expected behavior

Icons should read aloud the title text fo the SVG icons to a screen reader. 

## Additional context

- https://github.com/AlaskaAirlines/auro-icon/issues/46

## Exit criteria

There should be no additional work required for auro-hyperlink. The work itself is on the auro icons library and the removal of `aria-hidden` by default. 

Once this element is updated with a reference to the updated icons lib, the readability of the icon's alt info by screen readers should be addressed.

**Comment by blackfalcon** [comment]
Once the following PR is released, this repo will need to be updated to link the updated dependency version. 

- https://github.com/AlaskaAirlines/Icons/pull/168

**Comment by blackfalcon** [comment]
Released 

- https://github.com/AlaskaAirlines/auro-hyperlink/releases/tag/v3.2.1
